packages/cozy-harvest-lib/test/components/KonnectorUpdateInfos.spec.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-env jest */
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import { KonnectorUpdateInfos } from 'components/infos/KonnectorUpdateInfos'
+
+// Default props
+const intents = {
+  redirect: jest.fn()
+}
+const konnector = {
+  slug: 'test-konnector'
+}
+const t = key => key
+
+const props = {
+  intents,
+  konnector,
+  t
+}
+
+describe('KonnectorUpdateInfos', () => {
+  it('should render', () => {
+    const component = shallow(<KonnectorUpdateInfos {...props} />).getElement()
+    expect(component).toMatchSnapshot()
+  })
+
+  it('should render as blocking', () => {
+    const component = shallow(
+      <KonnectorUpdateInfos {...props} isBlocking={true} />
+    ).getElement()
+    expect(component).toMatchSnapshot()
+  })
+
+  it('should render button busy and disabled after click', () => {
+    const wrapper = shallow(<KonnectorUpdateInfos {...props} />)
+    wrapper.instance().handleClick()
+    wrapper.update()
+    expect(wrapper.getElement()).toMatchSnapshot()
```

Comment:
Not blocking, but I think this sort of test shouldn't be a snapshot. It would be easier to maintain if we tested the busy and disabled props, something like `expect(wrapper.dive().props.busy).toBe(true)`, instead of having to refer to a snapshot in a different file.

Reply:
You're totally right, I plaid guilty.